### ISSUE TITLE
flutterfire_ui fire this error when phone auth click on verify after safety net page (capcha is complete then click verify)

### ISSUE THREAD

**Issue by richard457** [firebase/flutterfire#8104]
```
Exception has occurred.
FlutterError (Looking up a deactivated widget's ancestor is unsafe.
At this point the state of the widget's element tree is no longer stable.
To safely refer to a widget's ancestor in its dispose() method, save a reference to the ancestor by calling dependOnInheritedWidgetOfExactType() in the widget's didChangeDependencies() method.)
```

**Comment by darshankawar** [comment]
@richard457 
Is this occuring using the plugin example ? https://github.com/FirebaseExtended/flutterfire/tree/master/packages/flutterfire_ui/example

Are you trying to Android or iOS ? Can you also provide your `flutter doctor -v` and steps to replicate ?

**Comment by richard457** [comment]
Still getting this issue don't know why it is closed without providing the solution. @darshankawar

**Comment by richard457** [comment]
> Hey @richard457. We need more information to resolve this issue but there hasn't been an update in 7 weekdays. I'm marking the issue as stale and if there are no new updates in the next 7 days I will close it automatically.
> 
> If you have more information that will help us get to the bottom of this, just add a comment!

Yes I am still experiencing the issue and I think this is not flutter fault lather the flutter_fire issue

**Comment by richard457** [comment]
> Since there haven't been any recent updates here, I am going to close this issue.
> 
> @richard457 if you're still experiencing this problem and want to continue the discussion just leave a comment here and we are happy to re-open this.

 Element._debugCheckStateIsActiveForAncestorLookup.<anonymous closure>
package:flutter/…/widgets/framework.dart:4153
#1      Element._debugCheckStateIsActiveForAncestorLookup
package:flutter/…/widgets/framework.dart:4167
#2      Element.dependOnInheritedWidgetOfExactType
package:flutter/…/widgets/framework.dart:4182
#3      FlutterFireUIActions.maybeOf
package:flutterfire_ui/…/auth/actions.dart:41
#4      FlutterFireUIActions.inherit
package:flutterfire_ui/…/auth/actions.dart:48
#5      startPhoneVerification.<anonymous closure>

**Comment by richard457** [comment]
@darshankawar please re-open this issue

**Comment by darshankawar** [comment]
Reopening.

**Comment by lesnitsky** [comment]
fixed in #9343
